docs/source/nlp/punctuation_and_capitalization.rst

```diff
@@ -0,0 +1,345 @@
+    - `roberta-base`, `roberta-large`, `distilroberta-base`
+    - `albert-base-v1`, `albert-large-v1`, `albert-xlarge-v1`, `albert-xxlarge-v1`, `albert-base-v2`, `albert-large-v2`, `albert-xlarge-v2`, `albert-xxlarge-v2`
+
+- classification head parameters:
+    - the number of layers in the classification heads (`model.punct_head.punct_num_fc_layers` and `model.capit_head.capit_num_fc_layers`)
+    - dropout value between layers (`model.punct_head.fc_dropout` and `model.capit_head.fc_dropout`)
+
+- optimizer (`model.optim.name`, for example, `adam`)
+- learning rate (`model.optim.lr`, for example, `5e-5`)
+
+
+Inference
+---------
+
+An example script on how to run inference on a few examples, could be found
+at `examples/nlp/token_classification/punctuation_capitalization_evaluate.py <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/token_classification/punctuation_capitalization_evaluate.py>`_.
+
+To run inference with the pre-trained model on a few examples, run:
+
+.. code::
+
+    python punctuation_capitalization_evaluate.py \
+           pretrained_model=<PRETRAINED_MODEL>
+
+Required Arguments for inference:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* :code:`pretrained_model`: pretrained PunctuationCapitalization model from list_available_models() or path to a .nemo file, for example: punctuation_en_bert or your_model.nemo
+
+
+Model Evaluation
+----------------
+
+An example script on how to evaluate the pre-trained model, could be found
+at `examples/nlp/token_classification/punctuation_capitalization_evaluate.py <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/token_classification/punctuation_capitalization_evaluate.py>`_.
+
+To run evaluation of the pre-trained model, run:
+
+.. code::
+
+    python punctuation_capitalization_evaluate.py \
+           model.dataset.data_dir=<PATH/TO/DATA/DIR>  \
+           pretrained_model=punctuation_en_bert \
+           model.test_ds.text_file=<text_dev.txt> \
+           model.test_ds.labels_file=<labels_dev.txt>
+
+
+Required Arguments:
+^^^^^^^^^^^^^^^^^^^
+* :code:`pretrained_model`: pretrained PunctuationCapitalization model from list_available_models() or path to a .nemo file, for example: punctuation_en_bert or your_model.nemo
+* :code:`model.dataset.data_dir`: Path to the directory that containes :code:`model.test_ds.text_file` and :code:`model.test_ds.labels_file`.
+
+
+Optional Arguments:
+^^^^^^^^^^^^^^^^^^^
+* :code:`model.test_ds.text_file` and :code:`model.test_ds.labels_file`: text_*.txt and labels_*.txt file names is the default text_dev.txt and labels_dev.txt from the config files should be overwritten.
+* Other :code:`model.dataset` or :code:`model.test_ds` arguments to override fields in the config file of the pre-trained model.
+
+
+During evaluation of the :code:`test_ds`, the script generates two classification reports: one for capitalization task and \
+another one for punctuation task. This classification reports include the following metrics:
+
```

Comment:
is there anything useful printed in the output to explain here? about the dataset processing part or evaluation metrics?